### PR TITLE
Docs for the new Dashboard

### PR DESCRIPTION
**Pull request by sylwiavargas**
# PR Description

### Summary of my changes and explanation of my decisions

- I've covered the different parts of the new Dashboard
- I've refreshed the screenshots and linked up different pages relating to the new Dashboard
- I've added a section about Collections

There's more work to be done but this will suffice for now given that there are subsequent releases planned.

### CODE REVIEW

**Comment by sulco** [comment]
A small one: the first screenshot in [Importing private projects](https://deploy-preview-114--stackblitz-docs.netlify.app/guides/user-guide/importing-projects#importing-private-projects) could be also updated

**Comment by sulco** [comment]
Looks good! The only thing I'd consider is putting the Collections section on its own page, just like we do with "Starter projects" – it would make for a less overwhelming "Getting started" page, and probably help with the findability (it's a distinct feature, but we can't find it in the User Guide's navigation). 

If we agree this is a good idea, it would be better to do it now rather than later, because this will be linked from the onboarding modal, so changing the URL will be a bit trickier then.

**Comment by sylwiavargas** [comment]
> Looks good! The only thing I'd consider is putting the Collections section on its own page, just like we do with "Starter projects" – it would make for a less overwhelming "Getting started" page, and probably help with the findability (it's a distinct feature, but we can't find it in the User Guide's navigation).
> 
> If we agree this is a good idea, it would be better to do it now rather than later, because this will be linked from the onboarding modal, so changing the URL will be a bit trickier then.

Done ✅

File: docs/guides/user-guide/getting-started.md [hunk 1]
```diff
@@ -10,41 +10,64 @@ head:
...
+## Importing code for quick demos
 
-No starter project for your favorite framework? Or maybe you'd like to import your own code?
+Alternatively, you can also use StackBlitz classic editor to [open an existing GitHub repository](/guides/user-guide/importing-projects). This option is suitable for the situations when there is no starter project for your favorite framework featured on the dashboard and you'd like to prepare a demo nevertheless.
 
-Worry not! With StackBlitz, you can [import projects](/guides/user-guide/importing-projects) from your own files or from GitHub repositories.
+## Grouping your projects into Collections
 
-## Using the StackBlitz editor
+If you'd like to group projects together, you could create a Collection. This feature is helpful when you like to create a list of your demos or inspirational projects you've forked, or can be used as your developer portfolio.
```
Comment (HeyGarrison) [inline docs/guides/user-guide/getting-started.md:51]:
```suggestion
If you'd like to group projects together, you could [create a collection](/guides/user-guide/collections). This feature is helpful when you like to create a list of your demos or inspirational projects you've forked or can be used as your developer portfolio.
```

**Review by HeyGarrison — APPROVED** [review]
Added in a tiny suggestion. Feel free to refactor it. :)

This looks great!

**Comment by sylwiavargas** [comment]
Thank you for checking it, @HeyGarrison! I know you're away today.

**Review by sulco — APPROVED** [review]
🤌

**Comment by sylwiavargas** [comment]
@sulco could you have a look now? 👀

**Comment by sulco** [comment]
> @sulco could you have a look now? 👀

Love it, so clear! 🤌 Thank you, @sylwiavargas! Let's merge it

**Pull request merged**